src/components/organisms/CreateProjectModal/CreateProjectModal.tsx

```diff
@@ -43,38 +43,56 @@ const CreateProjectModal: React.FC = () => {
   const [isModalHid, setIsModalHid] = useState(false);
   const [isSubmitEnabled, setSubmitEnabled] = useState(false);
   const [isFormValid, setIsFormValid] = useState(false);
+  const [isEditingRootPath, setIsEditingRoothPath] = useState(false);
   const [pickedPath, setPickedPath] = useState(projectsRootPath);
 
   const {openFileExplorer, fileExplorerProps} = useFileExplorer(
     ({folderPath}) => {
       if (folderPath) {
         setPickedPath(folderPath);
-        createProjectForm.setFields([{name: 'rootFolder', value: folderPath, errors: ['Path exists!']}]);
-        setSubmitEnabled(false);
+        setFormValues({...formValues});
+        setIsEditingRoothPath(false);
       }
     },
     {isDirectoryExplorer: true}
   );
 
-  const setProjectRootPath = (rootFolder: string) => {
-    const pathExists = existsSync(rootFolder);
-    setFormValues({...formValues, rootFolder});
+  const setProjectRootPath = () => {
+    let projectPath = formValues.rootFolder;
+    if (!isEditingRootPath) {
+      projectPath = formValues.name ? path.join(pickedPath, formValues.name.toLowerCase()) : pickedPath;
```

Comment:
suggest to also replace space with '-', i.e. something in line with  `path.join(pickedPath, formValues.name.toLowerCase().replace(' ', '-' )`